.ci/scripts/update_ci_branches.py

```diff
@@ -10,9 +10,12 @@
 branches = safe_load(config)["ci_update_branches"]
 
 github_api = "https://api.github.com"
-workflow_path = "/actions/workflows/publish_images.yaml/dispatches"
-url = f"{github_api}/repos/pulp/pulp-oci-images{workflow_path}"
 
 for branch in branches:
     print(f"Updating {branch}")
+    if ( branch >= 3.22 ):
```

Comment:
@dkliban I just realized I need to handle "latest". If I were to reverse it and check if branch < 3.22, would that cause "latest" to be treated like "3.22"?

Reply:
Never mind, I know how to fix this now.